udata_hydra/analysis/csv.py

```diff
@@ -103,7 +105,13 @@ async def analyse_csv(check_id: int = None, url: str = None, file_path: str = No
     exception_file = str(check.get("resource_id", "")) in exceptions
 
     headers = json.loads(check.get("headers") or "{}")
-    tmp_file = open(file_path, "rb") if file_path else await download_resource(url, headers, exception_file)
+    tmp_file = open(file_path, "rb") if file_path else await download_resource(
+        url=url,
+        headers=headers,
+        # do we want to detect the format here?
```

Comment:
I don't think so. But, what we actually want here is a max size we want to download here. Using CSV's one is good with me for debug.